docs_user/modules/proc_deploying-the-bare-metal-provisioning-service.adoc

```diff
@@ -173,8 +174,8 @@ spec:
 * After applying the configuration update to {rhos_acro}, apply the configuration and start the related services. The {bare_metal} begins to poll power state of the bare metal nodes:
```

Comment:
Has the TODO comment been addressed in the Verification section?

Reply:
I think this TODO needs addressing. I would suggest two steps:

1. Verify the list of endpoints (Followed by command to do list endpoints)
2. Verify the baremetal ndoes (Followed by command to list nodes)